i18n/react-intl/en.json

```diff
@@ -6,15 +6,13 @@
   "index_manifesto_content2": "We need an online culture guided by a spirit of transparency and civility, built by and for humans. More Human Internet aims to provide a space for citizens of the web to be heard. We believe the existence of this online space will change how websites are built and how humans interact with them.",
   "index_manifesto_content3": "Let's create a More Human Internet.",
   "index_aboutus_header": "About Us",
-  "index_aboutus1": "More Human Internet is a nonprofit organization building a free, public platform that facilitates dialog between the users and maintainers of the web to promote civility, transparency, and the greater good.",
-  "index_aboutus2": "We are enlisting like-minded web citizens to help spearhead this effort.",
-  "index_aboutus3": "Our beta will allow website maintainers to get direct feedback from users, starting a productive and human conversation. Try the ",
-  "index_aboutus4": "demo",
-  "index_aboutus5": " to learn more.",
+  "index_aboutus1": "More Human Internet is a non-profit organization of digital activists using web technologies to promote civility, transparency, and the greater good.",
+  "index_aboutus2": "Our first product is Roar!, a free, open source browser extension that automatically captures a snapshot of any online issue and addresses a tweet to the site's maintainer.",
```

Comment:
Sorry, last minute copy updates. Instead of `Our first product is` use "Our first application is`

Also, no comma after `Roar!`.

On that note, this should be a link to the extension (as it is in the Google Doc) So you may need to break this apart into multiple JSON entries to do that.

Reply:
All changed as instructed.